.github/workflows/pull-request-workflow.yml

```diff
@@ -31,14 +35,10 @@ jobs:
 
   lint-commit:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [16.x]
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-          configFile: .commitlintrc.json
```

Comment:
because for actions/checkout@v3 configFile isn't valid input 
<img width="521" alt="image" src="https://user-images.githubusercontent.com/110382066/205080664-6ae34258-718d-4485-9d30-c46729e56f05.png">

Reply:
Okay, but are you sure that now commintlint job is using our actual config file? Does it pick automatically?

Reply:
oh, right. I got it, thanks